src/postgres/contrib/yb_auh/yb_auh.c

```diff
@@ -80,7 +80,7 @@ circularBufferIndex *CircularBufferIndexArray = NULL;
 static int circular_buf_size = 0;
 static int circular_buf_size_kb = 16*1024;
 
-static int auh_sampling_interval = 1;
+static int auh_sampling_interval = 1000;
```

Comment:
it would be good to add a `_ms` prefix to help the reader with what this is supposed to represent.